app/code/Magento/Customer/Test/Unit/Model/ForgotPasswordToken/GetCustomerByTokenTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Customer\Test\Unit\Model\ForgotPasswordToken;
+
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Customer\Api\Data\CustomerInterface;
+use Magento\Customer\Api\Data\CustomerSearchResultsInterface;
+use Magento\Customer\Model\ForgotPasswordToken\GetCustomerByToken;
+use Magento\Framework\Api\SearchCriteria;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Exception\State\ExpiredException;
+use Magento\Framework\Phrase;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test GetCustomerByToken class
+ */
+class GetCustomerByTokenTest extends TestCase
+{
+    protected const RESET_PASSWORD = 'resetPassword';
+
+    /**
+     * @var SearchCriteriaBuilder|MockObject
+     */
+    protected $searchCriteriaBuilderMock;
+
+    /**
+     * @var SearchCriteria|MockObject
+     */
+    protected $searchCriteriaMock;
+
+    /**
+     * @var CustomerRepositoryInterface|MockObject
+     */
+    protected $customerRepositoryMock;
+
+    /**
+     * @var CustomerSearchResultsInterface|MockObject
+     */
+    protected $searchResultMock;
+
+    /**
+     * @var CustomerInterface|MockObject
+     */
+    protected $customerMock;
+
+    /**
+     * @var GetCustomerByToken;
+     */
+    protected $model;
+
+    /**
+     * @inheritDoc
+     */
+    public function setUp()
```

Comment:
```suggestion
    protected function setUp()
```
`setUp` method is `protected`. There is no reason to change it's visibility (refer: `\PHPUnit\Framework\TestCase::setUp`)